typings/audit.d.ts

```diff
@@ -23,6 +23,8 @@ declare global {
 
     export type ScoringModeValue = Audit.ScoringModes[keyof Audit.ScoringModes];
 
+    export type DisplayValue = string | Array<string|number>
```

Comment:
there might be an even more elegant way to do this, but you can do
```ts
interface DisplayValueArray extends Array<string | number> {
  0: string;
}

export type DisplayValue = string | DisplayValueArray;
```
to enforce a string in the first position (and a length of at least 1)

Reply:
done 👍